src/sentry/static/sentry/app/components/group/externalIssuesList.tsx

```diff
@@ -16,37 +14,60 @@ import SentryAppInstallationStore from 'app/stores/sentryAppInstallationsStore';
 import SentryAppComponentsStore from 'app/stores/sentryAppComponentsStore';
 import ExternalIssueStore from 'app/stores/externalIssueStore';
 import ErrorBoundary from 'app/components/errorBoundary';
+import {
+  Group,
+  Project,
+  Organization,
+  PlatformExternalIssue,
+  Event,
+  SentryAppComponent,
+  SentryAppInstallation,
+  GroupIntegration,
+} from 'app/types';
+
+type Props = AsyncComponent['props'] & {
+  group: Group;
+  project: Project;
+  organization: Organization;
+  event: Event;
+};
+
+type State = AsyncComponent['state'] & {
+  components: SentryAppComponent[];
+  sentryAppInstallations: SentryAppInstallation[];
+  externalIssues: PlatformExternalIssue[];
+  integrations: GroupIntegration[];
+};
+
+class ExternalIssueList extends AsyncComponent<Props, State> {
+  unsubscribables: any[] = [];
 
-class ExternalIssueList extends AsyncComponent {
   static propTypes = {
-    api: PropTypes.object.isRequired,
     group: SentryTypes.Group.isRequired,
     project: SentryTypes.Project.isRequired,
     organization: SentryTypes.Organization.isRequired,
     event: SentryTypes.Event,
-    orgId: PropTypes.string,
   };
 
-  getEndpoints() {
+  getEndpoints(): [string, string][] {
     const {group} = this.props;
     return [['integrations', `/groups/${group.id}/integrations/`]];
   }
 
-  constructor(props) {
-    super(props);
-    this.unsubscribables = [];
-    this.state = {
+  constructor(props: Props) {
+    super(props, {});
+    this.state = Object.assign({}, this.state, {
       components: [],
       sentryAppInstallations: [],
       externalIssues: [],
-    };
+    });
   }
 
   componentWillMount() {
     super.componentWillMount();
 
     this.unsubscribables = [
-      SentryAppInstallationStore.listen(this.onSentryAppInstallationChange),
+      SentryAppInstallationStore.listen(this.onSentryAppInstallationChange, this),
```

Comment:
What is the second parameter for and why do the other stores not need it?

Reply:
@billyvg You need to pass the context in. The reason the other stores do not need it is because they aren't TS yet. I probably should just pass `this` for the other stores as well.

![Screen Shot 2019-10-23 at 8 48 11 AM](https://user-images.githubusercontent.com/8533851/67411090-eeeb0500-f571-11e9-841f-bceeb4e8d52a.png)